src/domain/callout/creation-dialog/step-layout/StepLayout.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FC, useMemo } from 'react';
```

Comment:
I think this whole file should be in a common place for it to be used by other wizards/stepped forms.
In fact I think we have already something

Reply:
It's a layout used for creating callouts. I don't expect it to be used from the outside.